packages/app-desktop/gui/TrashNotification/TrashNotification.tsx

```diff
@@ -51,11 +51,15 @@ export default (props: Props) => {
 			notification.remove();
 			void onCancelClick(props.lastDeletion);
 		};
-		const notification = popupManager.createPopup(() => (
-			<TrashNotificationMessage message={msg} onCancel={handleCancelClick}/>
-		), { type: NotificationType.Success });
-		notification.scheduleDismiss();
-	}, [props.lastDeletion, props.dispatch, popupManager]);
+		const notification = notify(
+			<TrashNotificationMessage message={msg} onCancel={handleCancelClick} />,
+			{
+				delay: 4000,
+				type: NotificationType.Success,
+				persistent: false,
+			},
+		);
+	}, [props.lastDeletion, props.dispatch, notify]);
```

Comment:
I can't help but notice that this actually the same amount of code, or maybe more than the original one. And this PR is meant to simplify notifications.

See the original issue:

> So we should make it possible to call a simple function that would display a notification. Maybe based on the existing plugin dialogs.showToast() function?

I think this is what we need